upload.go

```diff
@@ -127,7 +909,7 @@ func processUpload(upReq UploadRequest) (upload Upload, err error) {
 	}
 
 	if len(extension) == 0 {
-		extension = "ext"
+		extension = extensions[mimetype]
```

Comment:
What if the mimetype isn't in the map? We want "ext" as fallback if the map doesn't contain it.